### DISCUSSION TITLE
My Github pages doesn't seem to be working please help.<title>

### DISCUSSION THREAD

**Discussion by Gideon1107** [community/community#116603] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I keep getting 
"""This site can’t provide a secure connection gideon1107.github.io sent an invalid response.

ERR_SSL_PROTOCOL_ERROR""" on all pages I deployed.  I tried clearing cache still not working. Tried on all browser still not working which made me believe it might be account itself.

**Comment by su-resh** [comment]
## GitHub pages problem.
### The issue you might be getting is due to incorrect formatting of the files. Make sure to make a index.html. You can refer to [pages](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site). Reading the docs you can solve the issue. If the error persists, share more details on it. Happ to help.

**Reply by Gideon1107** [reply]
yes, I have index.html file. but still not working

**Comment by Adword07** [comment]
> ## GitHub pages problem.
> ### The issue you might be getting is due to incorrect formatting of the files. Make sure to make a index.html. You can refer to [pages](https://alightmotionsapps.com/). Reading the docs you can solve the issue. If the error persists, share more details on it. Happ to help.
Thank you for highlighting the issue.

**Reply by su-resh** [reply]
## Can you share the screenshot of the page action?

**Reply by Gideon1107** [reply]
![Capture](https://github.com/community/community/assets/51706462/a1df4379-24a0-4e32-a375-cd0eb028c6ad)

Link to the github repo >> [https://github.com/Gideon1107/My-Resume](url)

**Comment by su-resh** [comment]
I guess the problem is occurring because your repository is private. GitHub
pages support hosting of private repositories for premium or educational id
only.

Try changing the repository into public and rerun the job.

**Comment by ghost** [comment]
your repository is private!!!!!!!!!!!!!!
